.github/workflows/testing_and_building_repo.yml

```diff
@@ -0,0 +1,110 @@
+name: testing_and_building_repo
+on: [push, pull_request]
+jobs:
+  build: 
+    strategy:
+      matrix:
+        os: ['ubuntu-18.04']
+
+    runs-on: ${{matrix.os}}
```

Comment:
Please use consistent whitespace conventions - we have `${{matrix.os}} ${ {matrix.os }} and ${{ matrix.os}}` below